regress/connection-timeout.sh

```diff
@@ -1,6 +1,12 @@
 #	$OpenBSD: connection-timeout.sh,v 1.2 2023/01/17 10:15:10 djm Exp $
 #	Placed in the Public Domain.
 
+if [ "$os" == "windows" ]; then
```

Comment:
We might want to add a pester test to this before we release

Reply:
per offline discussion, leaving that as backlog item